### PR TITLE
Support for original output coloring

### PR DESCRIPTION
**Pull request by to-bar**
Note: Azure Pipelines doesn't support ANSI colors for Releases yet (https://github.com/microsoft/azure-pipelines-agent/issues/1569#issuecomment-618505636) therefore `--no-color` option or `NO_COLOR` env should be used.

### CODE REVIEW

**Review by seriva — DISMISSED** [review]

File: ci/pipelines/linters.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ pool:
 
 variables:
   ansible_lint_error_threshold: 338
-  pylint_score_cli_threshold: 9.44
+  pylint_score_cli_threshold: 9.51
```
Comment (przemyslavic) [inline ci/pipelines/linters.yaml]: please also add `python3 -m` before calling `pip` module in lines 37 and 70

**Review by przemyslavic — APPROVED** [review]

**Review by seriva — APPROVED** [review]

**Pull request merged**